recipes/pymt_ecsimplesnow/meta.yaml

```diff
@@ -0,0 +1,50 @@
+{% set name = "pymt_ecsimplesnow" %}
+{% set version = "0.2.2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/pymt-lab/{{ name }}/archive/v{{ version }}.tar.gz
+  sha256: 8ee53a9981d9c208b8464cedd058ca302d2dd09afe14c9eba7b153bf6f0961c6
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed --no-cache-dir -vvv"
+  skip: True  # [win]
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('fortran') }}
+    - bmi-fortran=1.2
```

Comment:
Is `bmi-fortran` supposed to be in all three types of requirements? Naively I would expect only `host` and `run`.
```suggestion
```